datashare-app/src/main/java/org/icij/datashare/Plugin.java

```diff
@@ -22,19 +22,28 @@
 
 public class Plugin extends Extension {
     private static final Pattern versionBeginsWithV = Pattern.compile("v[0-9.]*");
+    public final List<String> extensions;
 
     @JsonCreator
     public Plugin(@JsonProperty("id") String id,
                   @JsonProperty("name") String name,
                   @JsonProperty("version") String version,
                   @JsonProperty("description") String description,
                   @JsonProperty("url") URL url,
-                  @JsonProperty("homepage") URL homepage){
+                  @JsonProperty("homepage") URL homepage,
+                  @JsonProperty("extensions") List<String> extensions
+    ){
         super(id, name, version, description, url, homepage, Type.PLUGIN);
+        this.extensions = extensions == null ? List.of() : extensions;
```

Comment:
We might use an Opionalable instead:

```suggestion
        this.extensions = ofNullable(extensions).orElse(List.of());
```